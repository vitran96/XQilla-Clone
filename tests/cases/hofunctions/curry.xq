import module namespace func = "http://snelson.org.uk/functions/functional";

let $f := func:curry(concat#5)
return $f("foo")(" bar")(" baz")(" what's")(" next?")