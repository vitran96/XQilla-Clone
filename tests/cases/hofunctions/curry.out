foo bar baz what's next?
